### ISSUE TITLE
Icon request: icon-skrill, icon-payby-skrill,

### ISSUE THREAD

**Issue by RetinaInc** [FortAwesome/Font-Awesome#4668]
Paypal is not legal in my country. The only option that works for online payments here is offered by Skrill ( previously moneybooker). I would like to add skrill payment gateways in some of my websites. Currently there is no icon for it in font awesome that i am using. So here i would like you to add this icon too.
![skrill-payby-85x47_en](https://cloud.githubusercontent.com/assets/3315212/4528906/1e4507a8-4d73-11e4-83a4-93ccb951c2d6.gif)

![skrill-85x37_en](https://cloud.githubusercontent.com/assets/3315212/4528916/362ebd14-4d73-11e4-9793-6aa04fef72d1.gif)

thank you

**Comment by RetinaInc** [comment]
so bad 4.3 don't have this icon but added whats already there like pinterest and facebook. I don't know why people are not requesting here but its obviously the most popular after paypal. Ecoomerce sites need this options. please try to add it soon.

**Comment by tagliala** [comment]
@RetinaInc unfortunately this request is not popular :\

You should consider a custom build: https://github.com/FortAwesome/Font-Awesome/wiki/Customize-Font-Awesome

**Comment by juliandormon** [comment]
I further this request! Skrill is a very popular web payment method similar to Paypal for EU countries. Please re-consider this one.

**Comment by RetinaInc** [comment]
@juliandormon Yes I don't know Why other developers are not asking for it. Many clients demanding Skrill in their sites.

**Comment by feyll** [comment]
Hello, this post is hold. Now is solve this?

**Comment by NesooseN** [comment]
Any news on this, it would be nice to add skrill icon also, our clients use it constantly .

**Comment by sensibleworld** [comment]
So... our backlog of brand requests has reached a point where even we can't pretend we're keeping up. Some of these issues were opened before I had gray hair. And I've had gray hair for a LONG time. On top of that, treating requests as issues has always made it way too hard to see the forest for the trees.

To fix this, we are now managing all brand requests in **GitHub Discussions**. No more +1s or thumbs-ups needed. There's an upvote system built-in.

We are closing existing brand requests as part of this transition, but **we are not saying no** to your request.

**If this brand request is still relevant for you, please re-open it in Discussions** (and feel free to include a link back to this issue):
-> https://github.com/FortAwesome/Font-Awesome/discussions/new?category=brand-requests

And listen. We know this is annoying. If you need to vent, here are some pre-written responses you can use when opening the new discussion:

* Cool cool cool. No worry at all. I love re-requesting a brand that's redesigned three times while this issue was open.
* If an issue closes after seven years and nobody responds, did it ever really exist?
* I see you have chosen violence today. Re-opening per instructions.
* What's next? I have to file a TPS report, too?
* I am filled with the fury of a thousand suns. Why do I have to re-open the discussion? Couldn't you have just bulk converted all of the issues to discussions? What? You couldn't? What do you mean Github removed that feature earlier this year? You'd have to manually click the "convert to discussion" button on thousands of issues, while also ensuring it gets added to the correct category? That's a bunch of bull-phooey.
* No biggie.
